app/helpers/locks_helper.rb

```diff
@@ -34,4 +34,14 @@ def resource_lock_icon(resource)
     text = (lock.warning? ? "#{warning_icon} Warning" : "#{lock_icon} Locked")
     content_tag :span, text.html_safe, class: "label label-warning", title: strip_tags(lock.summary)
   end
+
+  def lock_affected(lock)
+    if lock.resource_type == "Stage"
+      "stage"
```

Comment:
why is this not a link / name of resource?

Reply:
it will only ever be shown on the page of the sage itself, so no need to link it to anything